packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -147,24 +146,24 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Gesture
   void initRenderView() {
     assert(renderView == null);
     renderView = RenderView(configuration: createViewConfiguration(), window: window);
-    renderView.prepareInitialFrame();
+    renderView!.prepareInitialFrame();
   }
 
   /// The object that manages state about currently connected mice, for hover
   /// notification.
-  MouseTracker get mouseTracker => _mouseTracker;
-  MouseTracker _mouseTracker;
+  MouseTracker get mouseTracker => _mouseTracker!;
+  MouseTracker? _mouseTracker;
 
   /// The render tree's owner, which maintains dirty state for layout,
   /// composite, paint, and accessibility semantics.
   PipelineOwner get pipelineOwner => _pipelineOwner;
-  PipelineOwner _pipelineOwner;
+  late PipelineOwner _pipelineOwner;
 
   /// The render tree that's attached to the output surface.
-  RenderView get renderView => _pipelineOwner.rootNode as RenderView;
+  RenderView? get renderView => _pipelineOwner.rootNode as RenderView?;
```

Comment:
renderView should be non-nullable, i think. We set it up in the constructor.

Reply:
It's nullable because of the following assert:

```
  void initRenderView() {
    assert(renderView == null);
```

Reply:
I made it non-nullable with a private `_isRenderViewInitialized`

Reply:
Sounds good.